back/migrations/20250203155216-remove_alt_field_from_provider_collection.ts

```diff
@@ -0,0 +1,13 @@
+//
+
+import type { Db } from 'mongodb';
+
+import type { IdentityProvider } from '@fc/identity-provider-adapter-mongo/schemas';
+
+//
+
+export async function up(db: Db) {
+  await db
+    .collection<IdentityProvider>('provider')
+    .updateMany({ alt: { $exists: true } }, { $unset: { alt: '' } });
+}
```

Comment:
question: does this change has already been applied in integration and production mongo database ?